meta-resin-common/recipes-core/initrdscripts/files/ro-rootfs

```diff
@@ -1,3 +1,7 @@
 #!/bin/sh
 
 bootparam_ro="true"
+
+ro_enabled() {
+    return 1
+}
```

Comment:
lgtm even though there are a few typos in the commit log. Won't explicitly mention them as I don't want my comments to appear I am blocking this PR